bubblewrap.c

```diff
@@ -499,9 +503,13 @@ privileged_op (int privileged_op_socket,
         die_with_error ("Can't mount proc on %s", arg1);
       break;
     case PRIV_SEP_OP_TMPFS_MOUNT:
-      if (mount ("tmpfs", arg1, "tmpfs", MS_MGC_VAL | MS_NOSUID | MS_NOEXEC, "mode=0755") != 0)
+    {
+      char *opt = label_mount("mode=0755", opt_file_label);
+      if (mount ("tmpfs", arg1, "tmpfs", MS_MGC_VAL | MS_NOSUID | MS_NOEXEC, opt) != 0)
```

Comment:
Not related to this change, but should we get rid of MS_MGC_VAL since it only applies to very old kernels?